app/controllers/exams_controller.rb

```diff
@@ -0,0 +1,61 @@
+class ExamsController < ApplicationController
+  before_action :set_exam, only: [:show, :edit, :update, :destroy]
+
+  # GET /exams
+  def index
+    @exams = Exam.all
+  end
+
+  # GET /exams/1
+  def show
+    :set_exam
+  end
+
+  # GET /exams/new
+  def new
+    @exam = Exam.new
+  end
+
+  # GET /exams/1/edit
+  def edit
+    :set_exam
```

Comment:
if i didn't call `:set_exam` again, will `edit` uses the same `id` all the time?